compiler-rt/include/profile/InstrProfData.inc

```diff
@@ -62,6 +62,7 @@
 #define INSTR_PROF_VISIBILITY
 #endif
 
+// clang-format off:re-enable clang-format after `issue #82426` is fixed
```

Comment:
is there a reason in the future to reenable clang format for these macros?

Reply:
reworded to explain the reason to turn off clang-format now (and when we might consider re-enable it). PTAL.